docs/cli/edgedb_instance/edgedb_instance_credentials.rst

```diff
@@ -0,0 +1,37 @@
+.. _ref_cli_edgedb_instance_credentials:
+
+
+===========================
+edgedb instance credentials
+===========================
+
+Display instance credentials (add `--json` for verbose).
```

Comment:
Oh right, also double backticks here as the test mentions

Reply:
Fixed https://github.com/edgedb/edgedb/pull/6686/commits/8a2fb7a5ea5ba929692243632968f3349fa6af53